### ISSUE TITLE
maybe an error in examples/3_NeuralNetworks/convolutional_network_raw.py

### ISSUE THREAD

**Issue by helloyide** [aymericdamien/TensorFlow-Examples#256]
in its maxpool2d(x, k) function, it calls the tf.nn.max_pool function with the parameter padding='SAME'. IMO padding 'SAME' is very rare in pool layer, 'VALID' should be better, just like the default value of other high level API: e.g. tf.layers.max_pooling2d, slim.layers.max_pool2d...
